clientv3/op.go

```diff
@@ -26,6 +26,7 @@ const (
 	tRange opType = iota + 1
 	tPut
 	tDeleteRange
+	tWatch
```

Comment:
should there be a watch type? it means an arbitrary Op can't go through a txn any more. I think watch can just use `tRange` here